helm/loki/values.yaml

```diff
@@ -17,10 +17,32 @@ multiTenantAuth:
   # -- See `kubectl explain deployment.spec.strategy` for more
   # -- ref: https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#strategy
   image:
+    # -- Repository to get multi-tenant proxy image from.
     repository: giantswarm/loki-multi-tenant-proxy-gs
-    tag: 0.3.0
+    tag: 0.4.0
     pullPolicy: IfNotPresent
     pullSecrets: []
+
+  configReloader:
+    image:
+      # -- Repository to get config reloader image from.
+      repository: giantswarm/configmap-reload
+      # -- Tag of image to use for config reloading.
+      tag: v0.9.0
+    # -- Resource requests and limits to apply to the config reloader containers.
+    resources:
+      requests:
+        cpu: "1m"
+        memory: "5Mi"
```

Comment:
Are those resources really enough ? Sure those containers won't need a lot to do their simple job but 1m CPU seems so small :sweat_smile:

Reply:
It's requests, not limits. So looks OK to me.